### ISSUE TITLE
Mouse Jiggler Stealth could be even more Stealthier

### ISSUE THREAD

**Issue by talktosanjay3** [DarkFlippers/unleashed-firmware#762]
### Describe the enhancement you're suggesting.

Thank you so much for all the wonderful contribution.
I was wondering if its possible to add Key strokes along with mouse movement.
Like,
1) Pressing Atl+Tab on certain intervals
2) Pressing random keys (except Enter)

**Comment by xMasterX** [comment]
yes this is possible, however random keys may cause random issues :)
need to find some "safe" keys like pressing media volume up and down to get it to previous level
or something like that
since pressing some random key is not a good thing if you got some stuff running, like some console app is doing its work in terminal, you left it for a while, jiggler jiggles and randomly sends ctrl+c command to that terminal, will be not fun

**Comment by talktosanjay3** [comment]
Completely agree!, It would work if the options are configurable and the user chooses what keystores he wants. 
Best Regards

**Comment by dreamcasting** [comment]
Can there also be an "infinite" jiggler modification while you're in there? A max of 30 minutes is great, but I have use cases needing much longer.

**Comment by mishamyte** [comment]
Looks like it's stale a bit!

If still actual, pls, open a fresh ticket with more detailed proposal.

Thanks!
